packages/sdk-react/src/components/approvalRequests/ApprovalRequestsTable/ApprovalRequestsTable.tsx

```diff
@@ -0,0 +1,290 @@
+            headerName: t(i18n)`Invoice #`,
+            sortable: false,
+            flex: 0.7,
+          },
+          {
+            field: 'counterpart_id',
+            headerName: t(i18n)`Counterpart`,
+            sortable: false,
+            flex: 1,
+            renderCell: (params) => (
+              <CounterpartCell counterpartId={params.value} />
+            ),
+          },
+          {
+            field: 'issued_at',
+            type: 'date',
+            headerName: t(i18n)`Issue date`,
+            sortable: false,
+            flex: 0.7,
+            valueFormatter: ({
+              value,
+            }: GridValueFormatterParams<
+              components['schemas']['PayableResponseSchema']['issued_at']
+            >) =>
+              value && i18n.date(value, DateTimeFormatOptions.EightDigitDate),
+          },
+          {
+            field: 'due_date',
+            type: 'date',
+            headerName: t(i18n)`Due date`,
+            sortable: false,
+            flex: 0.7,
+            valueFormatter: ({
+              value,
+            }: GridValueFormatterParams<
+              components['schemas']['PayableResponseSchema']['due_date']
+            >) =>
+              value && i18n.date(value, DateTimeFormatOptions.EightDigitDate),
+          },
+          {
+            field: 'status',
+            headerName: t(i18n)`Status`,
+            sortable: false,
+            flex: 0.7,
+            renderCell: (params) => (
+              <ApprovalRequestStatusChip status={params.value} />
+            ),
+          },
+          {
+            field: 'amount',
+            headerName: t(i18n)`Amount`,
+            sortable: false,
+            flex: 0.5,
+            valueGetter: (params) => {
+              const payable = params.row;
+
+              return payable.amount_to_pay && payable.currency
+                ? formatCurrencyToDisplay(
+                    payable.amount_to_pay,
+                    payable.currency
+                  )
+                : '';
+            },
+          },
+          {
+            field: 'created_by',
+            headerName: t(i18n)`Added by`,
+            sortable: false,
+            flex: 1,
+            renderCell: ({ value }) => <UserCell entityUserId={value} />,
+          },
+          {
+            field: 'actions',
+            renderHeader: () => null,
+            sortable: false,
+            flex: 0.5,
+            align: 'right',
+            renderCell: (params) => {
+              if (
+                params.row.status === ApprovalRequestStatus.WAITING &&
+                user?.id &&
+                params.row.user_ids.includes(user.id) &&
+                !params.row.approved_by?.includes(user.id)
+              ) {
+                return (
+                  <Stack direction="row" spacing={1}>
+                    <ApproveButton approvalRequestId={params.row.id} />
+                    <RejectButton approvalRequestId={params.row.id} />
```

Comment:
Unfortunately, we can't be sure that our customisers are even going to process Approve or Reject actions on parties. For `InvoicesTable` we used a special hook [`useInvoiceRowActionMenuCell()`](https://github.com/team-monite/monite-sdk/blob/94ad73e35649255c7a07152e901a6417d21afad9/packages/sdk-react/src/components/receivables/InvoicesTable/useInvoiceRowActionMenuCell.tsx#L18-L54) which takes `onRowActionClick()` and `rowActions` as an input from the [`InvoicesTable`](https://github.com/team-monite/monite-sdk/blob/94ad73e35649255c7a07152e901a6417d21afad9/packages/sdk-react/src/components/receivables/InvoicesTable/InvoicesTable.stories.tsx#L31-L45)

Reply:
By the way, the need to add such customization is debatable. If the `ApprovalRequest` state machine is simple, `onReject` and `onApprove` may be enough to display actions.

@nekedos, what do you think about this?

Reply:
> onReject and onApprove may be enough to display actions.

I agree it is enough. According to the documentation, there is no other action in the `/approval_requests`.

<img width="570" alt="Screenshot 2024-06-28 at 12 40 23" src="https://github.com/team-monite/monite-sdk/assets/368559/313f0ed3-7605-45fe-aab7-c40cffe3cf44">